report/requirements.txt

```diff
@@ -1,7 +1,7 @@
 plotly==5.18.0
 kaleido==0.2.1
-pandas==2.1.4
-matplotlib==3.8.2
+pandas==2.0.3
+matplotlib==3.7.5
```

Comment:
I'm curious about this downgrade

Reply:
I had Python 3.10 on my PC, which couldn't install the old dependencies (Python hell), so I downgraded to 3.8 because of some deps that I use. I think that last year I probably used 3.9 or something? Not much point in finding sense in Python package management..